src/main/resources/data/nucleoid_extras/lang/en_us.json

```
@@ -370,6 +370,8 @@
   "text.nucleoid_extras.statistics.waiting": "Waiting for statistics data",
   "text.nucleoid_extras.seconds": "%s second(s)",
 
+  "text.nucleoid_extras.ui.action.more": "Right-Click for more...",
```

I don't think 'click' should be capitalized here:

```suggestion
  "text.nucleoid_extras.ui.action.more": "Right-click for more...",
```